### PR TITLE
162161084 fix windows symlink

### PR DESCRIPTION
**Pull request by xtreme-jason-smith**

### CODE REVIEW

**Comment by xtreme-jason-smith** [comment]
Note: This will break the BOSH Agent as it requires BOSH Utils changes. There is a pull request to update the BOSH Utils with this PR: https://github.com/cloudfoundry/bosh-utils/pull/52

File: agent/applier/bundlecollection/file_bundle_windows_test.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	"os"
+
+	. "github.com/cloudfoundry/bosh-agent/agent/applier/bundlecollection"
+	"github.com/cloudfoundry/bosh-agent/agent/applier/bundlecollection/fakes"
+	fakefileutil "github.com/cloudfoundry/bosh-utils/fileutil/fakes"
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	fakesys "github.com/cloudfoundry/bosh-utils/system/fakes"
+)
+
+//go:generate counterfeiter -o fakes/fake_clock.go ../../../vendor/code.cloudfoundry.org/clock Clock
```
Comment (dpb587-pivotal) [inline agent/applier/bundlecollection/file_bundle_windows_test.go]: Is this something different than https://github.com/cloudfoundry/bosh-agent/tree/master/vendor/code.cloudfoundry.org/clock/fakeclock?

Comment (xtreme-jason-smith) [reply agent/applier/bundlecollection/file_bundle_windows_test.go]: Nope, it should be the same one. We copied part of the code from the file_bundle_test.go

**Comment by xtreme-jason-smith** [comment]
I forgot to mention that this commit does not include the go dep changes to pull in the BOSH Util code. Will that be taken care of automatically or should I update the pull request to include the dep changes?

**Pull request merged**